core/src/main/java/org/elasticsearch/index/mapper/core/NumberFieldMapper.java

```
@@ -178,6 +181,15 @@ public boolean useTermQueryWithQueryString() {
         public boolean isNumeric() {
             return true;
         }
+
+        @Override
+        public ValueFormat valueFormat(@Nullable String format, DateTimeZone timeZone) {
+            if (format == null) {
+                return new ValueFormat.Raw();
+            } else {
+                return new ValueFormat.Decimal(format);
+            }
```

should we throw an exception if timeZone is specified here?